eternalcore-core/src/main/java/com/eternalcode/core/feature/playtime/PlaytimeCommand.java

```diff
@@ -0,0 +1,55 @@
+package com.eternalcode.core.feature.playtime;
+
+import com.eternalcode.annotations.scan.command.DescriptionDocs;
+import com.eternalcode.core.injector.annotations.Inject;
+import com.eternalcode.core.notice.NoticeService;
+import com.eternalcode.core.util.DurationUtil;
+import dev.rollczi.litecommands.annotations.argument.Arg;
+import dev.rollczi.litecommands.annotations.command.Command;
+import dev.rollczi.litecommands.annotations.context.Context;
+import dev.rollczi.litecommands.annotations.execute.Execute;
+import dev.rollczi.litecommands.annotations.permission.Permission;
+import org.bukkit.Statistic;
+import org.bukkit.entity.Player;
+
+import java.time.Duration;
+
+@Command(name = "playtime", aliases = {"play-time", "pt"})
+@Permission("eternalcore.playtime")
+public class PlaytimeCommand {
+
+    private final NoticeService noticeService;
+
+    @Inject
+    public PlaytimeCommand(NoticeService noticeService) {
+        this.noticeService = noticeService;
+    }
+
+    @Execute
+    @DescriptionDocs(description = "Show your playtime")
+    void self(@Context Player player) {
+        this.noticeService.create()
+            .notice(translation -> translation.playtime().self())
+            .placeholder("{PLAYTIME}", this.formatPlaytime(player))
+            .player(player.getUniqueId())
+            .send();
+    }
+
+    @Execute
+    @Permission("eternalcore.playtime.other")
+    @DescriptionDocs(description = "Show playtime of a player", arguments = "<player>")
```

Comment:
the command show**s**
```suggestion
    @DescriptionDocs(description = "Shows playtime of a player", arguments = "<player>")
```